projects/hslayers/src/components/core/event-bus.service.ts

```diff
@@ -162,5 +162,9 @@ export class HsEventBusService {
   layerSelectedFromUrl: BehaviorSubject<VectorLayer> = new BehaviorSubject(
     null
   );
+  /**
+   * Cancels any external url data request from datasources panel
+   */
+  cancelUrlRequest: Subject<void> = new Subject<void>();
```

Comment:
Maybe better would be to put it in add-data.service ? To group these subjects by topic.

Reply:
Implemented the requested change.